### ISSUE TITLE
Régimen de capital CFDI v4

### ISSUE THREAD

**Issue by TheSpectroMx** [eclipxe13/CfdiUtils#94]
Hola! una duda.... hay algún método implementado en la librería para quitar el régimen de capital cuando se genera un CFDI v4.0 ? Muchas gracias de antemano.

**Comment by eclipxe13** [comment]
Tienes algún listado de los regímenes de capital (siglas y descripción) para poder crear un método para quitarlos?

**Comment by TheSpectroMx** [comment]
No, no lo tengo. De hecho creo que no hay como un catálogo pero deja investigo y doy un update.

**Comment by uziel200** [comment]
Sus respectivas descripciones:

| Siglas | Descripción |
| ------ | ------ |
| AC | Asociación Civil |
| SA | Sociedad Anónima |
| SA DE CV | Sociedad Anónima de Capital Variable |
| SA DE CV SOFOM ENR | Sociedad Anónima de Capital Variable Sociedades Financieras de Objeto Múltiple Entidad Regulada |
| SAB | Sociedad Bursátil |
| S DE RL | Sociedad de Responsabilidad Limitada |
| S DE RL DE CV | Sociedad de Responsabilidad Limitada de Capital Variable |
| SAS | Sociedad por Acciones Simplificadas |
| S EN C | Sociedad en Comandita |
